pipsi.py

```diff
@@ -82,14 +83,22 @@ def real_readlink(filename):
     return normpath(realpath(join(dirname(filename), target)))
 
 
+SubprocessResult = namedtuple('SubprocessResult', ('returncode', 'out', 'err'))
+
+
 def statusoutput(argv, **kw):
```

Comment:
the changed behaviour asks for an new name

Reply:
Good point! Any suggestions?

I'd lean towards maybe changing this to match  the [Python 3.5 subprocess](https://docs.python.org/3/library/subprocess.html#subprocess.run) signatures somewhat closer, e.g. `def run(argv, **kw)` and `RunResult(returncode, stdout, stderr)` (this is closer to `subprocess.CompletedProcess` but that one also has `args` as first item, I'm not sure about introducing that as it would break other call sites). What do you think?

Reply:
either a thing wrapper with the old name for the other call sites or a whole fix can do the solving
im not sure whats the best solution in the case of pipsi